test/mpi/comm/ic1.c

```diff
@@ -42,6 +42,27 @@ int main(int argc, char *argv[])
         }
 
         MPI_Comm_free(&intercomm);
+
+#if MPI_VERSION >= 4
+        /* Test MPI_Intercomm_create_from_groups */
+        MPI_Group self_group, world_group;
+        MPI_Comm_group(MPI_COMM_SELF, &self_group);
+        MPI_Comm_group(MPI_COMM_WORLD, &world_group);
```

Comment:
MPI says the input groups need to be disjoint, so I think this is not a correct example.

Reply:
I see. The remote_group is redundant information then, since it should be identical to the local_group on the remote processes. I'll make the updates.